pkgs/tools/package-management/protontricks/default.nix

```diff
@@ -55,6 +56,10 @@ buildPythonApplication rec {
 
   pythonImportsCheck = [ "protontricks" ];
 
+  passthru.updateScript = nix-update-script {
+    attrPath = pname;
+  };
+
```

Comment:
The default updater for `buildPythonApplication` doesn't work because it expects a stable release: https://github.com/NixOS/nixpkgs/blob/bc25e8f24f317daadaa8b19ed3d5c1937ae9cb69/pkgs/development/interpreters/python/update-python-libraries/update-python-libraries.py#L208-L210

 but protontricks is only released through tags: https://github.com/Matoking/protontricks/tags.